homeassistant/components/onvif/models.py

```diff
@@ -0,0 +1,59 @@
+"""ONVIF models."""
+from typing import List
+
+import attr
+
+
+@attr.s
+class DeviceInfo:
+    """Represent device information."""
+
+    manufacturer = attr.ib(type=str, default=None)
+    model = attr.ib(type=str, default=None)
+    fw_version = attr.ib(type=str, default=None)
+    mac = attr.ib(type=str, default=None)
```

Comment:
Preferred way since we're on a modern Python:

```suggestion
    manufacturer: str = attr.ib(default=None)
    model: str = attr.ib(default=None)
    fw_version: str = attr.ib(default=None)
    mac: str = attr.ib(default=None)
```

btw, there are nowadays also a similar feature in Python called data classes.